docs_app/content/external-resources.md

```diff
@@ -0,0 +1,13 @@
+# External Resources
+
+This is a collection of resources that can be found to help with the learning process of RxJS. The list is ever growing which will make it a place for you to come to for finding helpful information.
```

Comment:
Maybe add a hint here, that RxJS isn't responsible for the content of the sites or something like that